frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -875,7 +875,7 @@ export function AdminReceiverDashboard() {
     }, []);
 
     return (
-        <section className="grid-container">
+        <article>
```

Comment:
Articles make more semantic sense than sections for our pages because they're self-contained bodies of content.  Updated a bunch of other pages as well, but we should keep this in mind going forward.